<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtFinderUi</class>
 <widget class="QWidget" name="QtFinderUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finder</string>
  </property>
  <property name="styleSheet">
   <string>
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="m_searchText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search in:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="m_searchOption">
     <property name="styleSheet">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Title/Instructions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ingredients</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tags</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_findBtn">
     <property name="styleSheet">
      <string/>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_clearButton">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>227</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
